aten/src/ATen/native/cpu/UpSampleKernelAVXAntialias.h

```diff
@@ -323,11 +323,13 @@ void upsample_avx_bilinear_uint8(
   std::vector<at::Tensor> horiz_indices_weights, vert_indices_weights;
   unsigned int horiz_weights_precision, vert_weights_precision;
 
-  bool needs_unpacking = (num_channels == 3 || num_channels == 4) && input.is_contiguous(at::MemoryFormat::ChannelsLast);
+  bool skip_unpacking = (num_channels == 3 || num_channels == 4) && \
+      input.is_contiguous(at::MemoryFormat::ChannelsLast) && \
+      output.is_contiguous(at::MemoryFormat::ChannelsLast);
```

Comment:
See other related suggestion below

```suggestion
  bool skip_unpacking = (num_channels == 3 || num_channels == 4) && \
      input.is_contiguous(at::MemoryFormat::ChannelsLast);
```